web/yo/app/scripts/controllers/gene.js

```diff
@@ -293,7 +294,7 @@ angular.module('oncokbApp')
                         if (reviewObjs[i].get('updateTime')) {
                             mostRecent = i;
                         }
-                    } else if (Date.parse(reviewObjs[mostRecent].get('updateTime')) < Date.parse(reviewObjs[i].get('updateTime'))) {
+                    } else if (reviewObjs[mostRecent].get('updateTime') < reviewObjs[i].get('updateTime')) {
```

Comment:
This worries me. What about the updateTime be the string appears again? Remember the complicated updateTime validate function? Can we reuse them?